package/rpm/HIRS.spec

```diff
@@ -164,6 +164,9 @@ if [ "$1" = "1" ]; then
     # open necessary ports
     sh /opt/hirs/scripts/common/firewall_configure_tomcat.sh
 
+    # Allow Tomcat to use port 3306 to communicate with MySQL
+    semodule -i /opt/hirs/extras/aca/tomcat-mysql-hirs.pp
```

Comment:
Based off discussion on phone call, perhaps we should have some check for SELinux enabled before running this command.

Reply:
I update this and tested it.